recipe/patch_yaml/urllib3.yaml

```
@@ -0,0 +1,10 @@
+# https://github.com/conda-forge/urllib3-feedstock/issues/84
+# urllib3 2.2.2 is not compatible with Python 3.7
+if:
+  name: urllib3
+  version: 2.2.2
```

It doesn't hurt to be a bit more specific about the artifact we are patching.

```suggestion
  version: 2.2.2
  build_number_in: [0]
```